store-implementation/accumulo-store/src/test/java/uk/gov/gchq/gaffer/accumulostore/utils/LegacySupportTest.java

```diff
@@ -52,125 +54,98 @@
  */
 public class LegacySupportTest {
 
-    @TempDir
-    static File tempDir;
+    File tempDir;
+    Configuration conf;
+
+    @BeforeEach
+    public void setUp() {
+        conf = new Configuration();
+        tempDir = Files.createTempDir();
+    }
+
+    @AfterEach
+    public void cleanUp() {
+        tempDir.deleteOnExit();
```

Comment:
This deletion method works because the file is no longer in use when it comes to be deleted, as the deletion occurs [only when the JVM exits](https://docs.oracle.com/javase/8/docs/api/java/io/File.html#deleteOnExit--), not during the tests. Instead of using before/after each, all could be used if each of the 4 tests that create a file used a different filename.